docs/open-source-project/TOC.md

```diff
@@ -42,6 +42,9 @@
 ### [FAQ](/open-source-project/reference/mic/mic-faq.md)
 
 # Release Notes
+## [Tizen 5.0 Public M2](/open-source-project/release-notes/tizen-5-0-m2.md)
+## [Tizen 5.0 Public M1](/open-source-project/release-notes/tizen-5-0-m1.md)
```

Comment:
@wtprince I think the order of **Tizen 5.0 Public M1** and **Tizen 4.0 Public M3** is changed.
The **Tizen 5.0 Public M1** (May. 31, 2018) is published before **Tizen 4.0 Public M3 (Aug. 31, 2018)**.
So **Tizen 5.0 Public M1** comes before **Tizen 4.0 Public M3**

Reference on Tizen Developer Site

![image](https://user-images.githubusercontent.com/9958374/46993340-ab351280-d149-11e8-937d-ef26546d39ea.png)

Reply:
will change the order as follows:
## [Tizen 5.0 Public M2](/open-source-project/release-notes/tizen-5-0-m2.md)
## [Tizen 4.0 Public M3](/open-source-project/release-notes/tizen-4-0-m3.md)
## [Tizen 5.0 Public M1](/open-source-project/release-notes/tizen-5-0-m1.md)
## [Tizen 4.0 Public M2](/open-source-project/release-notes/tizen-4-0-m2.md)
...